components/features/features.module.scss

```diff
@@ -1,9 +1,13 @@
-@import '../variables';
-@import '../mixins';
+@import '../../styles/variables';
+@import '../../styles/mixins';
 
 .features {
   margin: 0 auto;
   max-width: $max-content;
+
+  @include breakpoint(medium-only) {
+    margin: 0 5rem;
```

Comment:
I'm glad you raised about this change - I should have mentioned about the intent behind this proposed change of margin size in the description. 

I've proposed this change as I noticed that the 2.5 rem margin becomes skewed to the left under the ipad dimensions - see below:

<img width="737" alt="image" src="https://user-images.githubusercontent.com/27775658/157012567-b4001d31-b690-486b-933b-54ccf69a6a56.png">

<img width="712" alt="image" src="https://user-images.githubusercontent.com/27775658/157012695-5864866f-6af8-4370-b239-b848d103fb42.png">

In terms of the spacing you mentioned here, I am guessing you are referring to the spacing between the tabs? One thing I noticed is that the spacing between the tabs are actually set up tighter in the desktop mode ( as seen below) , which this new set up also matches the spacing between the tabs in the desktop mode, hence I am not entirely sure if this is a problem. 

<img width="1510" alt="image" src="https://user-images.githubusercontent.com/27775658/157013532-a19ac040-bd70-43ec-b1fc-c508e68d7e22.png">

All in all, the main reasoning behind this is that I think the 2.5 rem border skews everything to the left, hence I am introducing a larger border to 'even' things out. It would be good to also confirm with design on the border since without the design to confirm, we are both kind of doing a guessing game here. I'll confirm with Mackay on this and et you know.

Reply:
As per agreed amongst us in the design catch up, I have not taken out the 5rem border and followed the 3.5 overall border instead.

Reply:
Excellent.

Would it be easier to add the margin back to the body element only and not each component, or would that create issues for the differing background colors?

Reply:
thanks! 
the margin is now applied to the body element, and instead I applied the negative margin for the case-studies section to eiminate the differing background colors.